README.md

```diff
@@ -59,16 +63,22 @@ Before installing the Telerik AppBuilder CLI, verify that your system meets the
 **Minimum Software Requirements**
 
 * OS X Mavericks
-* Node.js 0.10.26 or a later stable official 0.10.x release except 0.10.34<br/>A [known issue](http://docs.telerik.com/platform/appbuilder/troubleshooting/known-issues/known-issues-cli-and-sp#the-appbuilder-command-line-interface-and-appbuilder-package-for-sublime-text-27-have-introduced-the-following-known-issues) prevents the AppBuilder CLI from working properly with Node.js 0.10.34.
+* Node.js 0.10.26 or a later stable official release except 0.10.34<br/>A [known issue](http://docs.telerik.com/platform/appbuilder/troubleshooting/known-issues/known-issues-cli-and-sp#the-appbuilder-command-line-interface-and-appbuilder-package-for-sublime-text-27-have-introduced-the-following-known-issues) prevents the AppBuilder CLI from working properly with Node.js 0.10.34.
 * An Internet browser (latest official release)
 
 **Additional Software Requirements for iOS On-Device Deployment**
 
 * iTunes (latest official)
+* For testing in the native emulator
+	* Xcode 5 or later
```

Comment:
Nope, nobody will follow it and check the sys reqs anyway. =)

As much as I'd like this refer-to-the-proper-source approach to work, it is not working. So what we need to do is put our sys reqs in one place where it's relevant for them to be.